handlers/product-move-api/src/main/scala/com/gu/productmove/SQS.scala

```diff
@@ -0,0 +1,136 @@
+package com.gu.productmove
+
+import com.gu.productmove.GuStageLive.Stage
+import com.gu.productmove.refund.RefundInput
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
+import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.{GetQueueUrlRequest, GetQueueUrlResponse, SendMessageRequest}
+import zio.*
+import zio.json.*
+
+trait SQS {
+  def sendEmail(message: EmailMessage): ZIO[Any, String, Unit]
+
+  def queueRefund(refundInput: RefundInput): ZIO[Any, String, Unit]
+}
+
+object SQS {
+  def sendEmail(message: EmailMessage): ZIO[SQS, String, Unit] = {
+    ZIO.environmentWithZIO(_.get.sendEmail(message))
+  }
+
+  def queueRefund(refundInput: RefundInput): ZIO[SQS, String, Unit] = {
+    ZIO.environmentWithZIO(_.get.queueRefund(refundInput))
+  }
+}
+
+object SQSLive {
+  val layer: ZLayer[AwsCredentialsProvider with Stage, String, SQS] =
+    ZLayer.scoped(for {
+      stage <- ZIO.service[Stage]
+      sqsClient <- initializeSQSClient().mapError(ex => s"Failed to initialize SQS Client with error: $ex")
+      emailQueueUrlResponse <- getEmailQueue(stage, sqsClient)
+      refundQueueUrlResponse <- getRefundQueue(stage, sqsClient)
+    } yield new SQS {
+      override def sendEmail(message: EmailMessage): ZIO[Any, String, Unit] =
+        for {
+          result <- ZIO
```

Comment:
nitpick: because it's not used
```suggestion
          _ <- ZIO
```